packages/typespec-azure-resource-manager/lib/operations.tsp

```diff
@@ -69,6 +71,7 @@ op ArmResourceListByParent<
  */
 @get
 @autoRoute
+@list
 @listsResource(Resource)
```

Comment:
I have a comment here: https://github.com/Azure/azure-rest-api-specs/pull/36014/files#r2221212583

Reply:
yeah need to check if we can put `@list` on arm `ListBy` templates

Reply:
checked and actually its because they are using a custom template with the same name, updating the pr to apply `@list` on the template instead